src/theme/stylus/page.styl

```diff
@@ -24,15 +22,30 @@
 }
 
 .content {
-    margin-left: auto
-    margin-right: auto
-    max-width: $content-max-width
+    position: relative
+    top: 0
+    bottom: 0
+    overflow-y: auto
+    right: 0
+    left: 0
+    padding: 0 15px
     padding-bottom: 50px
 
+    main {
+        margin-left: auto
+        margin-right: auto
+        max-width: $content-max-width
+    }
+
     a {
         text-decoration: none;
         &:hover { text-decoration: underline; }
     }
 
     img { max-width: 100%; }
 }
+
+.sidebar-visible .content {
+    position: absolute
+    top: 52px
```

Comment:
Not sure how well this will scale - the header may have a different size depending on font settings (it's `51px` on my machine).

Reply:
It's 52 on mine haha.

Reply:
Can't you define it as something like `3 em` to make sure the size scales appropriately? I imagine hard-coding the size is probably going to make a difference when people view things on mobile.